drivers/scmi-msg/scmi_pinctrl.h

```diff
@@ -0,0 +1,197 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (c) 2023, EPAM Systems
+ */
+#ifndef SCMI_MSG_PINCTRL_H
+#define SCMI_MSG_PINCTRL_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <lib/utils_def.h>
+
+#define SCMI_PROTOCOL_VERSION_PINCTRL 0x10000U
+
+#define SCMI_PINCTRL_ATTR_NAME_SZ 16U
+#define SCMI_PINCTRL_NAME_MAX_SZ  64U
+
+#define ALIGN_NEXT(VALUE, INTERVAL)                                            \
```

Comment:
There is already `round_up()` from `utils_def.h`, no need for own implementation